h2integrate/tools/eco/utilities.py

```diff
@@ -1,3679 +0,0 @@
-        solar_plant_power = np.array(hybrid_plant.pv.generation_profile[0:simulation_length])
-        output.update({"pv generation [kW]": solar_plant_power})
-    if hybrid_plant.wind:
-        wind_plant_power = np.array(hybrid_plant.wind.generation_profile[0:simulation_length])
-        output.update({"wind generation [kW]": wind_plant_power})
-    if hybrid_plant.wave:
-        wave_plant_power = np.array(hybrid_plant.wave.generation_profile[0:simulation_length])
-        output.update({"wave generation [kW]": wave_plant_power})
-    if hybrid_plant.battery:
-        battery_power_out_mw = hybrid_plant.battery.outputs.P
-        output.update(
-            {"battery discharge [kW]": [(int(p > 0)) * p for p in battery_power_out_mw]}
-        )  # convert from MW to kW and extract only discharging
-        output.update(
-            {"battery charge [kW]": [-(int(p < 0)) * p for p in battery_power_out_mw]}
-        )  # convert from MW to kW and extract only charging
-        output.update({"battery state of charge [%]": hybrid_plant.battery.outputs.dispatch_SOC})
-    total_generation_hourly = hybrid_plant.grid._system_model.Outputs.system_pre_interconnect_kwac[
-        0:simulation_length
-    ]
-    output.update({"total generation hourly [kW]": total_generation_hourly})
-    output.update(
-        {
-            "total generation curtailed hourly [kW]": hybrid_plant.grid.generation_curtailed[
-                0:simulation_length
-            ]
-        }
-    )
-    output.update({"total accessory power required [kW]": solver_results[0]})
-    output.update({"grid energy usage hourly [kW]": solver_results[1]})
-    output.update({"desal energy hourly [kW]": [solver_results[2]] * simulation_length})
-    output.update(
-        {
-            "electrolyzer energy hourly [kW]": electrolyzer_physics_results[
-                "power_to_electrolyzer_kw"
-            ]
-        }
-    )
-    output.update({"electrolyzer bop energy hourly [kW]": solver_results[5]})
-    output.update(
-        {"transport compressor energy hourly [kW]": [solver_results[3]] * simulation_length}
-    )
-    output.update({"storage energy hourly [kW]": [solver_results[4]] * simulation_length})
-    output.update(
-        {
-            "h2 production hourly [kg]": electrolyzer_physics_results["H2_Results"][
-                "Hydrogen Hourly Production [kg/hr]"
-            ]
-        }
-    )
-    if "hydrogen_storage_soc" in h2_storage_results:
-        output.update({"hydrogen storage SOC [kg]": h2_storage_results["hydrogen_storage_soc"]})
-    if "hydrogen_demand_kgphr" in h2_storage_results:
-        output.update({"hydrogen demand [kg/h]": h2_storage_results["hydrogen_demand_kgphr"]})
-
-    df = pd.DataFrame.from_dict(output)
-
-    filepath = output_dir / "data/production"
-
-    if not filepath.exists():
-        filepath.mkdir(parents=True)
-
-    df.to_csv(filepath / "energy_flows.csv")
-
-    return output
-
-
-def calculate_lca(
```

Comment:
Add to dropped features - LCA. I'm a little concerned about dropping this, but maybe it is just not of interest right now. @jrannoni ?